src/apps/activities/api/activities.py

```diff
@@ -215,6 +215,63 @@ async def applet_activities_for_target_subject(
     )
 
 
+async def applet_activities_for_respondent_subject(
+    applet_id: uuid.UUID,
+    subject_id: uuid.UUID,
+    user: User = Depends(get_current_user),
+    language: str = Depends(get_language),
+    session=Depends(get_session),
+    answer_session=Depends(get_answer_session),
+) -> ResponseMulti[ActivityOrFlowWithAssignmentsPublic]:
+    applet_service = AppletService(session, user.id)
+    await applet_service.exist_by_id(applet_id)
+
+    await SubjectsService(session, user.id).exist_by_id(subject_id)
+
+    # Restrict the endpoint access to owners, managers, coordinators, and assigned reviewers
+    await CheckAccessService(session, user.id).check_subject_subject_access(applet_id, subject_id)
+
```

Comment:
Just a little wrench to throw in here 🔧, but I think it's easily addressed. Everything else is perfect.

Limited account aren't eligible to be respondents. So if it's discovered that the respondent subject is a limited account, then we can skip a lot of the logic in here to check if they have assignments or have submitted answers on behalf of another subject – neither is possible for limited accounts.

Currently with the logic in here, if you query for activities/flows against a limited account as a respondent, it returns a list of activities/flows that are set to `auto_assign = true`, which isn't correct.

I think you can just do a check here and short-circuit past a bunch of the below code for limited accounts.